codespell_lib/data/dictionary.txt

```diff
@@ -31780,15 +31874,19 @@ spile->spite, spiral,
 spiltting->splitting
 spindel->spindle
 spindels->spindles
+spined->spun, spines,
```

Comment:
Is "spined" correct as an adjective? https://www.merriam-webster.com/dictionary/spine

Reply:
Removed.